src/content/pages/volunteers.mdx

```diff
@@ -13,16 +13,20 @@ We will also have a small number of perks for you, some small gifts to say "than
 
 # Things to remember
 
-* This is in person volunteering and would require you to be present at the venue.
+* This is **in-person** volunteering and would require you to be present at the venue.
 * While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
-* To enter the conference you need a ticket.
-* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.
+* To enter the conference you need a ticket. Some volunteers are eligible for a free ticket, see perks below.
+* You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. We appreciate any help you can give us.
+* You can get a free combined ticket (Tutorials + Conference) if you pledge to volunteer for ten (10) hours or more.
 
 # Perks for volunteers
 
-* Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)
-* Be part of a great team and make new friends.
-* A special group photo with all volunteers (time will be announced on the #onsite-volunteers on discord)
+* A free combined ticket (Tutorials + Conference) if you volunteer for ten (10) hours or more!
```

Comment:
Maybe even put this at the end of the list. - But that is just my current preference at this moment.